config.ini

```diff
@@ -0,0 +1,17 @@
+; Schulich Delta Hermes Specification File
+
+forwardPeriod=500
+
+[JsonFormat]
+packetTitle=Gen5
+
+[SerialPort]
+portName=/dev/pts/7
+baudrate=9600
```

Comment:
Baudrate we're using in the telemetry stack is 115200, it's written in the protocol data sheet in cell H29 if a reference is needed.